utils/scheduler.js

```diff
@@ -171,6 +176,38 @@ const merchantMonitor = async (discordClient) => {
   }
 };
 
+
+/**
+ * Cronjob to Monitor WoW Server Status
+ * @param {Client} discordClient
+ */
+const wowServerMonitor = (client) => {
+  schedule.scheduleJob('*/1 * * * *', async () => {
+    try {
+
+      //Frostmourne realm id is 3725
+      const wowrealmstatus = await wowrealmStatus(3725)
+
+      if(wowServerStatus == ""){
```

Comment:
in JS best practices `===` usually preferred for performance and also adheres to strict typing

more here:
https://codeahoy.com/javascript/2019/10/12/==-vs-===-in-javascript/